package/src/views/useTouchHandler.ts

```diff
@@ -3,10 +3,7 @@ import { useCallback, useRef } from "react";
 import { PixelRatio } from "react-native";
 
 import type {
-  ExtendedTouchInfo,
-  TouchHandlers,
-  TouchHandler,
-  TouchInfo,
+  ExtendedTouchInfo, TouchHandler, TouchHandlers, TouchInfo
```

Comment:
Ideally we don't want changes in formatting and ordering in a PR, so this block could be left out since it is not really related to the fix. We'll leave it for now, just wanted to let you know :)